lamindb_setup/_connect_instance.py

```diff
@@ -304,7 +304,10 @@ def connect(instance: str | None = None, **kwargs: Any) -> str | tuple | None:
                 else:
                     from lamindb_setup.core.django import reset_django
 
-                    if settings.instance.slug != "none/none":
+                    if (
+                        settings._instance_exists
```

Comment:
`settings.instance` is not available if `settings._instance_exists` is `False`, hence we need to add this check.